cli/command/network/connect.go

```diff
@@ -1,12 +1,14 @@
 package network
 
 import (
+	"fmt"
 	"github.com/docker/cli/cli"
 	"github.com/docker/cli/cli/command"
 	"github.com/docker/cli/opts"
 	"github.com/docker/docker/api/types/network"
 	"github.com/spf13/cobra"
 	"golang.org/x/net/context"
+	"strings"
```

Comment:
Put `fmt` and `strings` first in the list, then a blank line before the non-stdlib imports.